src/components/Book.js

```diff
@@ -1,19 +1,28 @@
 import PropTypes from 'prop-types';
 
-const Book = ({ book }) => (
-  <tr>
-    <td>{book.id}</td>
-    <td>{book.title}</td>
-    <td>{book.category}</td>
-  </tr>
-);
+const Book = ({ book, removeBook }) => {
+  const handleClick = () => {
+    removeBook(book);
+  };
```

Comment:
**Heads-Up**
I don't think you need to do this.
You already implement this in `BookList`

After using destructuring to get the `removeBook`
Just attach it to `onClick` on line 13.
that is;  `<button type="button" onClick={removeBook}>Remove Book</button>`